### PR TITLE
Update Bitcoin currency to use proper symbol, add mBTC and μBTC

### PR DESCRIPTION
**Pull request by overcookedpanda**
This is an update to the changes I submitted in https://github.com/grafana/grafana/pull/13125
Based on https://github.com/grafana/grafana/pull/13125#issuecomment-477138503 and https://github.com/grafana/grafana/issues/8564 :)

### CODE REVIEW

**Comment by overcookedpanda** [comment]
Please don't mark as stale :)

**Comment by phemmer** [comment]
While this may be the more accurate representation of the symbol, it does not render well on many systems. For example mine. While the old symbol is fine.
image

**Comment by overcookedpanda** [comment]
Ah, I also believe that's why I used the current symbol in the first PR I submitted, but it seemed to render properly on the commit, however, I did notice the same issue inside my IDE when I was making this, but figured it would render properly in modern browsers.

image

File: devenv/dev-dashboards-without-uid/panel_tests_polystat.json [hunk 1]
```diff
@@ -426,8 +426,16 @@
               "value": "currencyPLN"
             },
             {
-              "text": "Bitcoin (฿)",
+              "text": "Bitcoin (B͈̎)",
               "value": "currencyBTC"
+            },
+            {
+              "text": "micro Bitcoin (mBTC)",
```
Comment (mckn) [inline devenv/dev-dashboards-without-uid/panel_tests_polystat.json:433]:
```suggestion
              "text": "Millibitcoin (mBTC)",
```

**Review by mckn — CHANGES_REQUESTED** [review]
Love the idea to add these to the code base but looks like you have mixed the units together. Please update the units to reflect this list: https://en.bitcoinwiki.org/wiki/Units

File: devenv/dev-dashboards/panel-polystat/polystat_test.json [hunk 2]
```diff
@@ -426,8 +426,16 @@
...
-              "text": "Bitcoin (฿)",
+              "text": "Bitcoin (B͈̎)",
               "value": "currencyBTC"
+            },
+            {
+              "text": "micro Bitcoin (mBTC)",
+              "value": "currencymBTC"
+            },
+            {
+              "text": "milli Bitcoin (μBTC)",
```
Comment (mckn) [inline devenv/dev-dashboards/panel-polystat/polystat_test.json:437]:
```suggestion
              "text": "Micro Bitcoin (μBTC)",
```

File: devenv/dev-dashboards/panel-polystat/polystat_test.json [hunk 3]
```diff
@@ -426,8 +426,16 @@
               "value": "currencyPLN"
             },
             {
-              "text": "Bitcoin (฿)",
+              "text": "Bitcoin (B͈̎)",
               "value": "currencyBTC"
+            },
+            {
+              "text": "micro Bitcoin (mBTC)",
```
Comment (mckn) [inline devenv/dev-dashboards/panel-polystat/polystat_test.json:433]:
```suggestion
              "text": "Milli Bitcoin (mBTC)",
```

File: packages/grafana-data/src/valueFormats/categories.ts [hunk 4]
```diff
@@ -114,7 +114,9 @@ export const getCategories = (): ValueFormatCategory[] => [
       { name: 'Czech koruna (czk)', id: 'currencyCZK', fn: currency('czk') },
       { name: 'Swiss franc (CHF)', id: 'currencyCHF', fn: currency('CHF') },
       { name: 'Polish Złoty (PLN)', id: 'currencyPLN', fn: currency('PLN') },
-      { name: 'Bitcoin (฿)', id: 'currencyBTC', fn: currency('฿') },
+      { name: 'Bitcoin (B͈̎)', id: 'currencyBTC', fn: currency('B͈̎') },
+      { name: 'micro Bitcoin (B͈̎)', id: 'currencymBTC', fn: currency('mBTC') },
+      { name: 'milli Bitcoin (B͈̎)', id: 'currencyμBTC', fn: currency('μBTC') },
```
Comment (mckn) [inline packages/grafana-data/src/valueFormats/categories.ts:119]:
```suggestion
      { name: 'Micro Bitcoin (B͈̎)', id: 'currencyμBTC', fn: currency('μBTC') },
```

Comment (mckn) [inline packages/grafana-data/src/valueFormats/categories.ts:119]:
```suggestion
      { name: 'Milli Bitcoin (B͈̎)', id: 'currencymBTC', fn: currency('mBTC') },
      { name: 'Micro Bitcoin (B͈̎)', id: 'currencyμBTC', fn: currency('μBTC') },
```

**Review by mckn — APPROVED** [review]
Great work! LGTM. Thanks for adding this @overcookedpanda 🙏

**Comment by mckn** [comment]
@overcookedpanda can you latest master into this branch so it can be merged?

**Comment by overcookedpanda** [comment]
> @overcookedpanda can you latest master into this branch so it can be merged?

Done. Thank you again.

**Comment by mckn** [comment]
> While this may be the more accurate representation of the symbol, it does not render well on many systems. For example mine. While the old symbol is fine.
> image
> 
> IIRC, this discussion came up when the currency was first proposed several years ago.

Great feedback! I have verified it in Safari, Chrome and Mozilla and it seems to be rendered correct so I think we can proceed with this one.

**Comment by phemmer** [comment]
> > While this may be the more accurate representation of the symbol, it does not render well on many systems. For example mine. While the old symbol is fine.
> > image
> > IIRC, this discussion came up when the currency was first proposed several years ago.
> 
> Great feedback! I have verified it in Safari, Chrome and Mozilla and it seems to be rendered correct so I think we can proceed with this one.

That screenshot was taken from Chrome.

**Comment by phemmer** [comment]
Oh, and this is what it looks like on my phone (Android 9)
image

**Comment by overcookedpanda** [comment]
In Chrome Version 83.0.4103.61 (Official Build) (64-bit) for Windows 10 I see the following:
image

Using Chrome Version 83.0.4103.88 on an iPhone I see the following:
image

Safari:
image

**Comment by mckn** [comment]
@torkelo @dprokop what to you guys think regarding the bitcoin sign?

**Comment by torkelo** [comment]
Think we will have to wait a while for this one as it looks wrong for me as well (Chrome 81, Ubuntu 18)

**Comment by mckn** [comment]
> Think we will have to wait a while for this one as it looks wrong for me as well (Chrome 81, Ubuntu 18)

what do you think about revering the bitcoin sign but merge the milli and micro BTC? @torkelo

**Comment by torkelo** [comment]
that will work

**Comment by phemmer** [comment]
Was doing some unicode stuff when I stumbled across the `₿` symbol. I found it odd that my browser was rendering this properly considering the recent discussion on this PR. But it appears that this is a completely different symbol than what was proposed. In fact, it seems this symbol is the "official" symbol as well (however official a decentralized currency with no owner can be), at least according to [wikipedia](https://en.wikipedia.org/wiki/Bitcoin).

Anyway, just mentioning in case you wanted to revive the issue.

**Comment by mckn** [comment]
> Was doing some unicode stuff when I stumbled across the `₿` symbol. I found it odd that my browser was rendering this properly considering the recent discussion on this PR. But it appears that this is a completely different symbol than what was proposed. In fact, it seems this symbol is the "official" symbol as well (however official a decentralized currency with no owner can be), at least according to [wikipedia](https://en.wikipedia.org/wiki/Bitcoin).
> 
> Anyway, just mentioning in case you wanted to revive the issue.

Interesting, maybe something worth opening a new PR on? Not sure how much this is used and how big of an issue it is.

**Comment by overcookedpanda** [comment]
> Was doing some unicode stuff when I stumbled across the `₿` symbol. I found it odd that my browser was rendering this properly considering the recent discussion on this PR. But it appears that this is a completely different symbol than what was proposed. In fact, it seems this symbol is the "official" symbol as well (however official a decentralized currency with no owner can be), at least according to [wikipedia](https://en.wikipedia.org/wiki/Bitcoin).
> 
> Anyway, just mentioning in case you wanted to revive the issue.

I confirmed that this symbol works with Brave on Linux which did not work last time, however, it seems iOS has some issue with this one, lol.

In the Mail app's preview I see the following, which looks good: 
image

However, once I open the PR and view I see that it does not render correctly:
image

I would think it would be best to keep using the Thai Baht symbol for the time being, and just check for support later and re-visit the issue.

**Pull request merged**